### ISSUE TITLE
Mulitple levels of tabs

### ISSUE THREAD

**Issue by kilasuit** [microsoft/terminal#18166]
### Description of the new feature

instead of a single horizontal layer for tabs please allow multiple horizontal layers for tabs too

**Comment by zadjii-msft** [comment]
I don't believe this is something we could trivially support, without the upstream WinUI TabView supporting it. Can you file that feature request over at https://github.com/microsoft/microsoft-ui-xaml/issues? Thanks!

**Comment by kilasuit** [comment]
Added via https://github.com/microsoft/microsoft-ui-xaml/issues/10143

**Comment by FireFistisDead** [comment]
Can you elaborate more about this problem @kilasuit

**Comment by kilasuit** [comment]
<img width="1035" height="471" alt="Image" src="https://github.com/user-attachments/assets/d5d136f2-cf2a-4354-8463-f756ccdf3f67" />

**Comment by FireFistisDead** [comment]
Got it thanks! @kilasuit

**Comment by TheColorman** [comment]
Seems to be a duplicate of #16912

**Comment by zadjii-msft** [comment]
Ooh excellent find, thanks!

/dup #16912
